en/snap/with-plugins.md

```diff
@@ -350,68 +395,90 @@ Customer receives email notification| Receive notification|  Check MAP menu sett
 </details>
 <hr><br><br>
 
-### Prestashop
+### PrestaShop
 <hr>
-Midtrans ❤️ Prestashop! Integrate your Prestashop store with Midtrans Snap payment gateway. We strive to make payments simple for both the merchant and customers. This plugin will allow online payment on your Prestashop store using various online payment channel.
+Midtrans ❤️ PrestaShop! Integrate your PrestaShop store with Midtrans Snap payment gateway. Midtrans strives to make payments simple for you and the customers. This plugin will allow online payment on your PrestaShop store using various online payment channels.
 
-#### Requirements:
-* Prestashop 1.6 & 1.7 or greater
-* PHP version 5.4 or greater
-* MySQL version 5.0 or greater
-* Midtrans plugin for PrestaShop [ [Github](https://github.com/veritrans/SNAP-Prestashop) | [Zip](https://github.com/veritrans/SNAP-Prestashop/archive/master.zip) ].
 
-#### Installation Process:
+#### Requirements
+Some of the requirements to continue with the integration process, are listed below.
+* PrestaShop 1.6 and 1.7 or later
+* PHP version 5.4 or later
+* MySQL version 5.0 or later
+* Midtrans plugin for PrestaShop [[Github](https://github.com/veritrans/SNAP-Prestashop)|[Zip](https://github.com/veritrans/SNAP-Prestashop/archive/master.zip)].
```

Comment:
> [[Github](https://github.com/veritrans/SNAP-Prestashop)|[Zip](https://github.com/veritrans/SNAP-Prestashop/archive/master.zip)]

Please do not remove the spacing from 
> [Github](https://github.com/veritrans/SNAP-Prestashop) | [Zip](https://github.com/veritrans/SNAP-Prestashop/archive/master.zip)

The spacing helps reader to know that there are 2 separate/different link, without spacing it looks like 1 same link.
This feedback applies to all the other parts in this page.